tests/accesscontrol/helper/helper.go

```diff
@@ -156,3 +181,247 @@ func DefineAndCreateServiceOnCluster(name, namespace string, port int32, targetP
+	if err != nil {
+		return false, fmt.Errorf("failed to get node %s: %w", nodeName, err)
+	}
+
+	return strings.Contains(node.Status.NodeInfo.KernelVersion, "+rt"), nil
+}
+
+// Deploys a Machineconfig to switch the kernel to realtime type on all th enodes of a MachineConfigPool.
+//
+// Warning: This is a disruptive operation that will reboot all the worker nodes one by one.
+func DeployRTKernelMachineConfig(mcpName string, mcName string, mcpLabels map[string]string, timeout time.Duration) error {
+	// Get the eco-goinfra client.
+	egiClient := globalhelper.GetEcoGoinfraClient()
+	if egiClient == nil {
+		return fmt.Errorf("eco-goinrfa client is not initialized")
+	}
+
+	// Create custom MachineConfig with kerneltype realtime.
+	mcBuilder := egiMco.NewMCBuilder(egiClient, mcName).WithKernelType("realtime")
+	if mcBuilder == nil {
+		return fmt.Errorf("failed to get MachineConfig builder")
+	}
+
+	// Add labels to MC so the MCP can match it.
+	for k, v := range mcpLabels {
+		mcBuilder.WithLabel(k, v)
+	}
+
+	glog.V(5).Infof("Creating MC=%s in the cluster.", mcName)
+	// Apply MachineConfig
+	_, err := mcBuilder.Create()
+	if err != nil {
+		return fmt.Errorf("failed to create MachineConfig: %w", err)
+	}
+
+	// MachineConfigPool
+	mcpBuilder := egiMco.NewMCPBuilder(egiClient, mcpName)
+	if mcpBuilder == nil {
+		return fmt.Errorf("failed to get MachineConfigPool builder")
+	}
+
+	// Wait for MCP to start updating: condition Updating=True
+	glog.V(5).Infof("MCP=%s: waiting %s for condition Updating to be True", mcpName, tsparams.McpStartTimeout)
+
+	err = mcpBuilder.WaitToBeInCondition(mcv1.MachineConfigPoolUpdating, corev1.ConditionTrue, tsparams.McpStartTimeout)
+	if err != nil {
+		return fmt.Errorf("failed waiting for MachineConfigPool to set condition Updating=True: %w", err)
+	}
+
+	// Wait for MCP to finish the update using the dynamic timeout as deadline.
+	glog.V(5).Infof("MCP=%s: waiting %s for condition Updating to be False", mcpName, timeout)
+
+	err = mcpBuilder.WaitToBeInCondition(mcv1.MachineConfigPoolUpdating, corev1.ConditionFalse, timeout)
+	if err != nil {
+		return fmt.Errorf("failed waiting for MachineConfigPool to set condition Updating=False: %w", err)
+	}
+
+	// Make sure everything went ok
+	if mcpBuilder.IsInCondition(mcv1.MachineConfigPoolDegraded) {
+		return fmt.Errorf("machineconfigpool appears as Degraded after machineconfig was applied")
+	}
+
+	return nil
+}
+
+// Removes the Machineconfig that switches the kernel to realtime type on all the nodes of a MachineConfigPool.
+//
+// Warning: This is a disruptive operation that will reboot all the worker nodes one by one.
+func RemoveRTKernelMachineConfig(mcpName string, mcName string, timeout time.Duration) error {
+	// Get the eco-goinfra client.
+	egiClient := globalhelper.GetEcoGoinfraClient()
+	if egiClient == nil {
+		return fmt.Errorf("eco-goinrfa client is not initialized")
```

Comment:
`eco-goinrfa` --> `eco-goinfra`

Reply:
Thanks. Fixed in https://github.com/redhat-best-practices-for-k8s/certsuite-qe/pull/1059/commits/adee779dcdbe6f3b0536704cd7743230ebf8d04d